src/tools/spin/spinquic.cpp

```diff
@@ -439,16 +430,13 @@ void Spin(LockableVector<HQUIC>& Connections, bool IsServer)
 
                 HQUIC Connection;
                 HQUIC Session = GetRandomFromVector(Sessions);
-                PRINT("MsQuic->ConnectionOpen(%p, ...) = ", Session);
```

Comment:
Instead of removing all the PRINT statements, can they be turned into ETW/WPP trace statements?

Reply:
We already have events for API calls in msquic.